docs/creating-sql-with-python.md

```diff
@@ -85,7 +94,10 @@ templates - instead, using the
 [template function library](https://github.com/smart-on-fhir/cumulus-library/blob/main/cumulus_library/template_sql/base_templates.py)
 you can provide arguments to these templates that will allow you to
 generate standard types of SQL tables, as well as using templates targeted for
-bespoke operations. 
+bespoke operations. But you :can: write study specific templates if you have
```

Comment:
nit: use `_can_` instead - we have actual typography here, we don't need hints-for-humans :smile: